src/select/SimpleSelect.js

```diff
@@ -686,7 +686,7 @@ class SimpleSelect extends Component {
       inputValue: this.props.inputValue,
       onBlur: this.onInputBlur,
       onChange: this.onInputChange,
-      onFocus: this.onInputFocus
+      onFocus: this.onInputFocus,
```

Comment:
nit: remove trailing comma

Reply:
agreed, please remove and we're good to merge.